extensions/amp-story/1.0/amp-story.css

```diff
@@ -875,8 +875,9 @@ amp-story .amp-video-eq,
 
 .i-amphtml-story-interactive-component {
   align-self: center;
-  width: auto !important;
+  width: auto;
```

Comment:
Is it intentional to drop the `!important`?

Reply:
Just was adding in the description:

> Also: width cannot be currently overriden by the creators, but it should. Removed !important from width so that they can specify width:320px. Currently they can set a max width if they enclose it in a div and set the width of that div, but it's not desired if they can just set it with a CSS attribute

While I was playing with the component making the stories, I found that I couldn't set the width of the component with a CSS rule, but it makes a lot of sense that users should be able to. There's a max and min width already that ensure it will be properly displayed